pkgs/top-level/python-packages.nix

```
@@ -692,6 +692,8 @@ in {
 
   python-utils = callPackage ../development/python-modules/python-utils { };
 
+  pythonparser = disabledIf (pythonAtLeast "3.7") (callPackage ../development/python-modules/pythonparser { });
```

```suggestion
  pythonparser = callPackage ../development/python-modules/pythonparser { };
```
Where do you get this info from? You should instead add `disabled = ...` inside the expression.

---

✓

---

> I documented the reason in the commit message

Please add a comment in the expression as you will have to squash the new commits into the initial one.

---

right, ok